src/components/form/web-forms-settings-confirmation.vue

```diff
@@ -30,27 +34,43 @@ except according to the terms contained in the LICENSE file.
           </router-link>
         </template>
       </i18n-t>
+      <div class="modal-actions">
+        <button class="btn btn-link" type="button" @click="$emit('hide')">
+          {{ $t('action.cancel') }}
+        </button>
+        <button type="button" class="btn btn-primary"
+          @click="setWebformsEnabled">
+          {{ $t('webformsConfirmation.useOdkWebForms') }}
+        </button>
```

Comment:
I see that `confirm.value` had an `awaitingResponse` property. That makes me think that we need a `<spinner>` here.